### ISSUE TITLE
Question about reproducing the Semi-supervised Learning experiment

### ISSUE THREAD

**Issue by 4-0-4-notfound** [amirbar/DETReg#39]
When i using this checkpoint as pretrain 

![image](https://user-images.githubusercontent.com/76674846/153593997-d7839a61-155f-44de-9412-e317f7782c6a.png)

and using these script to reproducing the Semi-supervised Learning experiment

![image](https://user-images.githubusercontent.com/76674846/153594051-50f3977b-5b68-4c53-87aa-2ed7358a9837.png)

the result turns out to be huge difference :

![image](https://user-images.githubusercontent.com/76674846/153594623-a2eda34c-ed98-4c6e-994e-0381a3fc805e.png)

Please help me, did i missing anything in reproducing ?

By the way, i can reproduce the full COCO result @45.5AP. So the conda env is probably right.

**Comment by amirbar** [comment]
Thanks for opening this issue! can you please share the config file `.sh` you are using? Mainly, are you training for 2000 epochs schedule without lr_drop, following this change?

![image](https://user-images.githubusercontent.com/6403606/154132527-beb4fc26-ef78-4386-b761-06906383028a.png)

**Comment by amirbar** [comment]
Thanks for opening the issue. Seems like I've forgotten to update the `lr_drop` schedule in the code, which leads to the learning rate dropping too early. I've created a [PR](https://github.com/amirbar/DETReg/pull/40) and I will test it soon to make sure the result is now reproduced. You can also build on this git branch in the meantime if you need to.

**Comment by BIGBALLON** [comment]
Hi, @amirbar, I can not reproduce the result even update the ``lr_drop``, are there any other changes we need to do?

**Comment by amirbar** [comment]
Thanks for the reminder @BIGBALLON. I'm running experiments to reproduce it now on my internal repo and based on this repo. I'm keeping this issue open for now, and I'll circle back if this indeed reproduces on my end and if yes will apply a fix.

**Comment by amirbar** [comment]
As a sanity check, I've partially trained from MSCoco checkpoint for 800 epochs using 1% MSCoco data. After 800 epochs, this model achieves 25.4 AP. I assume that with more careful choice of checkpoint/more training it will improve to 26 AP like the reported result. 

That's the running command I've used:
`export GPUS_PER_NODE=8 && bash ./tools/run_dist_launch.sh 8 ./configs/DETReg_fine_tune_1pct_coco.sh --batch_size 1 --pretrain pretrained/checkpoint_coco.pth --output_dir exps/coco1pct_fromcoco --eval_every 200`

Here's the log file in case it is helpful:
[log.txt](https://github.com/amirbar/DETReg/files/8130180/log.txt)

Hope this helps!

**Comment by BIGBALLON** [comment]
@amirbar thanks for your helpful reply. it would be better to update the parameters in the readme.

**Comment by luoyq6** [comment]
> 当我使用此检查点作为预训练时
> 
> ![image](https://user-images.githubusercontent.com/76674846/153593997-d7839a61-155f-44de-9412-e317f7782c6a.png)
> 
> 并使用这些脚本重现半监督学习实验
> 
> ![image](https://user-images.githubusercontent.com/76674846/153594051-50f3977b-5b68-4c53-87aa-2ed7358a9837.png)
> 
> 结果证明是巨大的差异：
> 
> ![image](https://user-images.githubusercontent.com/76674846/153594623-a2eda34c-ed98-4c6e-994e-0381a3fc805e.png)
> 
> 请帮助我，我在复制时遗漏了什么吗？
> 
> 顺便说一句，我可以重现完整的 COCO 结果@45.5AP。所以康达环境可能是对的。
Excuse me, hello. Why do I get the error shown in the picture when I use the command to evaluate? Could you give me some tips?
![b39205ed6c3af39c07dc62491331797](https://user-images.githubusercontent.com/110521846/218689798-2625367d-fa8b-4463-ac86-60b20bc7fc06.png)
